packages/typespec-ts/test/integration/payloadPageable.spec.ts

```diff
@@ -0,0 +1,67 @@
+import { assert } from "chai";
+import PageableClientFactory, {
+  PageableClient
+} from "./generated/payload/pageable/src/index.js";
+
+describe("Pageable Client", () => {
+  let client: PageableClient;
+
+  beforeEach(() => {
+    client = PageableClientFactory({
+      allowInsecureConnection: true,
+      retryOptions: {
+        maxRetries: 0
+      }
+    });
+  });
+
+  it("should get pagable list without skiptoken", async () => {
+    try {
+      const result = await client
+        .path("/payload/pageable")
```

Comment:
Please refer this for paging cases

https://github.com/Azure/autorest.typescript/blob/main/packages/autorest.typescript/test/rlcIntegration/pagingRest.spec.ts#L26-L40

Reply:
I have updated this test file as you said.